cmd/githubgen/go.mod

```diff
@@ -4,11 +4,13 @@ go 1.22.0
 
 require (
 	github.com/google/go-github/v64 v64.0.0
-	go.opentelemetry.io/collector/confmap v1.15.1-0.20240911162712-6c2697c4453c
-	go.opentelemetry.io/collector/confmap/provider/fileprovider v1.15.1-0.20240911162712-6c2697c4453c
+	go.opentelemetry.io/collector/confmap v1.15.1-0.20240916093651-6029f3176705
+	go.opentelemetry.io/collector/confmap/provider/fileprovider v1.15.1-0.20240916093651-6029f3176705
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+require github.com/rogpeppe/go-internal v1.12.0 // indirect
```

Comment:
Can you share why this dependency is required? I wouldn't have expected a new dependency to be required when the only change is updating existing dependencies, but one of them must require it now?